tex/default_template.tex

```diff
@@ -0,0 +1,27 @@
+\documentclass[a4paper,10pt]{{letter}}
+\usepackage[utf8]{{inputenc}}
+\usepackage[freepin,{edge_type}]{{ticket}}
+\usepackage{{graphicx}}
+
+\unitlength=1mm
+\ticketSize{{{badge_width}}}{{{badge_height}}}
+\ticketNumbers{{{cards_per_page_x}}}{{{cards_per_page_y}}}
+
+\renewcommand{{\ticketdefault}}{{
```

Comment:
single curly brackets

Reply:
Here is it by design there are triple `{}` for the variables? Should they be double?

Reply:
Yeap, single, look for comparison: https://gist.github.com/dokato/fcf2445c16135802c4404f8a90bbd22a